contracts/utils/CurvedInterestRate.sol

```diff
@@ -1,51 +1,68 @@
-// SPDX-License-Identifier: Unlicense
+/// SPDX-License-Identifier: Unlicense
 pragma solidity >=0.8.4;
 
 import "prb-math/contracts/PRBMathUD60x18.sol";
 import "../interfaces/ICurvedInterestRate.sol";
+import "./Guardian.sol";
 
-contract CurvedInterestRate is ICurvedInterestRate {
+contract CurvedInterestRate is ICurvedInterestRate, Guardian {
```

Comment:
re-order the inheritance to place PausableGuardin_0_8 first then ICurvedInterestRate. It has no difference in functionality but looks cleaner as PausableGuardian is a contract and carries state storing capabilities.

Reply:
fixed